example/lib/main.dart

```diff
@@ -323,6 +359,70 @@ class _PrettyQrSettingsState extends State<_PrettyQrSettings> {
               },
             ),
           ),
+        if (widget.onExportPressed != null) ...[
+          const Divider(),
+          ListTile(
+            leading: const Icon(Icons.save_alt_outlined),
+            title: const Text('Export'),
+            onTap: () async {
+              await widget.onExportPressed?.call(imageSize);
+              const snackBar = SnackBar(
+                content: Text('Saved'),
+              );
+
+              ScaffoldMessenger.of(context).showSnackBar(snackBar);
```

Comment:
nit: analyzer warning: "Do not use BuildContexts across async gaps."

Reply:
fixed